README.rst

```diff
@@ -75,9 +75,46 @@ From System Package Manager
 ---------------------------
 
 If your operating system packages repolib, you can install it by running::
-    
+
     sudo apt install python3-repolib
 
+If your distro does not yet supply the package, you can install the
+[latest release](https://github.com/pop-os/repolib/releases/) using the
+steps below (At the time of writing this, the version is 2.0.0):
+
+    # Initialize some useful variables
+    PKG=python3-repolib_2.0.0_all.deb
+    URL="https://github.com/pop-os/repolib/releases/download/2.0.0/$PKG"
+
+    PREREQ_PKG=(curl python3-gnupg python3-debian ca-certificates)
+    . <(grep -E '^(VERSION_)?ID=' /etc/os-release)
+
+    # Make sure we're using the right mix, debian requires some missing
```

Comment:
`debian` should be capitalized in a comment.

I would split this comment's line after the comma so the next line isn't just one word. Also, change the comma to a semicolon since these are both independent clauses.

You used "you" above and "we" here; it might be good to make that consistent.

Reply:
Fixed the captialization and line length. As for You vs We, documentation has You, code comments has We. If this still bugs you, I'll modify this.

Reply:
> As for You vs We, documentation has You, code comments has We.

That's fine.